src/tests/fs_tests.rs

```diff
@@ -4034,6 +4034,33 @@ pub mod fs_tests {
 
         assert_eq!(cage.close_syscall(fd), 0);
         assert_eq!(cage.exit_syscall(EXIT_SUCCESS), EXIT_SUCCESS);
+
+        lindrustfinalize();
+    }
+
+    pub fn ut_lind_fs_shmget_syscall(){
+        // acquire locks and start env cleanup
+        let _thelock = setup::lock_and_init();
+        let cage = interface::cagetable_getref(1); 
+
+        let key = 33123;
```

Comment:
can we add a simple test case for passing IPC_PRIVATE?

Reply:
Thanks for pointing that out - have added the test!